index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Weather Report</title>
+    <link rel="stylesheet" href="styles/index.css">
+</head>
+<body>
+    <header class="header_header">
+        <span><h1>Weather Report</h1> </span>
+        <span> for the lovely city of </span>
+        <span> ✨</span>
+        <span id="header-city"> Seattle</span>
+        <span> ✨</span>
+    </header>
+    <main>
+        <section id="input-containers">
+            <section id="temp-container">
+                <h2>Temperture</h2>
+                <section id="change-temp-container">
+                    <span id="temp-buttons"><button id="increaseButton">⬆️</button><button id="decreaseButton">⬇️</button></span>
+                    <span><h2 id="temp">67</h2></span>
+                </section>
+            </section>
+            <section id="sky-container"> 
+                <h2>Sky</h2>
+                <select id="sky-select">
+                    <option value=1 >Sunny</option>
+                    <option value=2 >Cloudy</option>
+                    <option value=3 >Rainy</option>
+                    <option value=4 >Snowy</option>
```

Comment:
Using more descriptive values would make your code more readable:

```suggestion
                    <option value="sunny">Sunny</option>
                    <option value="cloudy">Cloudy</option>
                    <option value="rainy">Rainy</option>
                    <option value="snowy">Snowy</option>
```